.github/workflows/tests.yml

```diff
@@ -41,7 +41,7 @@ jobs:
 
     - name: Install test dependencies
       run: |
-        sudo apt-get install libgirepository1.0-dev python3-virtualenv nginx-full
```

Comment:
Was `python3-virtualenv` ever used acutally?

Reply:
It was described in the README.rst. It is not used in the workflow.

Reply:
I've added that bit of info to the commit message.